packages/core/src/common/test/test-logger.ts

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
I think this kind of situation will happen in other places too and I would like to avoid changing code for the tests only. It would allow for more tests to be written without refactoring the code too.

Reply:
I think it's a good idea to make your code independent of the infrastructure so it can not only be easier used in other scenarios (e.g. tests) but also be easier tested itself.

Reply:
I don't agree, if you generalize that practice you'll get abstract versions of every class only so the tests can mock some functions.
It should be refactored only if needed by actual code.